### DISCUSSION TITLE
How do I stop the serp_goog displaying my API key to console?

### DISCUSSION THREAD

**Discussion by omiq** [eliasdabbas/advertools#281] [Q&A] [answered]
When Serp_Goog is called it is printing info to console, including my API key?

```2023-05-17 13:29:31,022 | INFO | serp.py:728 | serp_goog | Requesting: q=magazine wordpress theme, cx=***********, key=**************```

How do I stop it doing that?

**Comment by omiq** [comment, marked as answer]
After digging into the code I found this seems to work in lieu of an official answer:

```
# Turn off logging
log_status = logging.getLogger().getEffectiveLevel()
logging.getLogger().setLevel(logging.CRITICAL)
```

**Reply by eliasdabbas** [reply]
Exactly. Glad that you found it. 
You can change the logging level that belongs to the `serp` module with a single command as follows:

```python
import advertools as adv

# change logging level:
adv.serp.set_logging_level(50)

# then run code normally without having it log anything
serp = adv.serp_goog(cx=cx, key=key, q='hotels in miami')
```
